components/form/autocompleted/src/index.js

```diff
@@ -143,15 +148,28 @@ export default class FormAutocompleted extends Component {
           {...this.props}
           handleSelect={this._handleSelect}
           active={active}
+          ref={node => { this.suggestList = node }}
           />
         )
       : null
   }
+  _handleOutsideClick = (event) =>
+    !this.excludeFromOutsideClick.includes(event.target) &&
+    this.setState({
+      showSuggestsList: false
+    })
 
   componentDidMount () {
     if (this.state.focus) {
       this.focusInput()
     }
+    this.excludeFromOutsideClick = [
```

Comment:
This solution is ok. But if another subelement were added, another ref should be created and added to excludeFromOutsideClick. Otherwise, we could have a very hardly detectable error. 

What about traversing up the target to see is one of the parents if the main node of the component ?
something like:

```js
var target = event.target
var isInnerClick = false
while(target.parentElement) {
   if( target === componentRef ) {
      isInnerClick = true
      break
   }
}

isInnerClick && this.setState({ showSuggestsList: false})
```

Outerclick could even be a component with this single responsibility